blockchain/blockchain.go

```diff
@@ -37,13 +45,26 @@ type Blockchain struct {
 	database db.DB
 }
 
-func NewBlockchain(database db.DB, network utils.Network) *Blockchain {
+func New(database db.DB, network utils.Network) *Blockchain {
 	return &Blockchain{
 		database: database,
 		network:  network,
 	}
 }
 
+func (b *Blockchain) Network() utils.Network {
+	return b.network
+}
+
+// StateCommitment returns the latest block state commitment.
+// If blockchain is empty zero felt is returned.
+func (b *Blockchain) StateCommitment() (commitment *felt.Felt, err error) {
```

Comment:
Will this be used for verifying state before storing the state update?

Reply:
No, this is just to make it available for through the blockchain object. I needed this mainly for testing.

Reply:
Alright. I just wanted to understand its use case.